### PR TITLE
collect console messages, show them at the end

### PR DESCRIPTION
**Pull request by enidemi**
**Problem:**
When running karma tests console messages with full stacktraces are in the middle of the output.

**Fix:**
Adding a custom reporter to show console messages after the test results. The messages are shortened and grouped by type. Patching mocha reporter to avoid having duplicated console messages.

**Commit Details:**
- added a patch to karma-mocha-reporter
- and a custom reporter
Screenshot 2021-10-01 at 16 10 55

### CODE REVIEW

**Comment by balazsbajorics** [comment]
can you share a screenshot? :)

**Comment by balazsbajorics** [comment]
it seems like pnpm busted you for changing the package.json but not changing the pnpm lock file

**Comment by enidemi** [comment]
> it seems like pnpm busted you for changing the package.json but not changing the pnpm lock file

oops :) fixed

**Review by balazsbajorics — APPROVED** [review]

**Review by seanparsons — APPROVED** [review]

File: editor/karma.conf.js [hunk 1]
```diff
@@ -19,8 +19,9 @@ module.exports = function (config) {
       'karma-chrome-launcher',
       'karma-viewport',
       'karma-mocha-reporter',
+      require('./test/karma-custom-reporter/short-console-messages'),
     ],
-    reporters: ['mocha'],
+    reporters: ['mocha', 'utopia'],
```
Comment (balazsbajorics) [inline editor/karma.conf.js:24]: nice

**Review by balazsbajorics — APPROVED** [review]

**Pull request merged**